cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -715,20 +715,21 @@ protected Service createService(String name, String type, List<ServicePort> port
         return service;
     }
 
-    protected Service createHeadlessService(List<ServicePort> ports, Map<String, String> annotations) {
+    protected Service createHeadlessService(List<ServicePort> ports, boolean publishNotReadyAddresses) {
```

Comment:
@tombentley because I don't see any kind of way for configure the `publishNotReadyAddresses` param which is always `true`, why do you add the parameter itself?